mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -205,6 +206,19 @@ public Future<JobExecution> updateBlocking(String jobExecutionId, JobExecutionMu
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantId) {
+    Promise<RowSet<Row>> promise = Promise.promise();
+    try {
+      String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
+      pgClientFactory.createInstance(tenantId).execute(query, promise);
+    } catch (Exception e) {
+      LOGGER.error("Error deleting jobExecution by ids {} : Exception : ", ids, e);
```

Comment:
It would be cleaner to remove "Exception : " from the log message string. You are not using the substitution token `{}`. The logger implementation we use appends the error message at the end, but it isn't always the case.

Reply:
I think have the log level as ERROR makes it easier. and also you could wrap all the ids in brackets like so `[ id1, id2 ]`. I dont think it would be too difficult to know where logging starts. Yes, please make the change.

Reply:
Hi @okolawole-ebsco - Updated changes as per review comment, Thanks 🙂